### PR TITLE
rework import

### PR DESCRIPTION
**Pull request by taulinger**
- [x]  move SourceType to /domain 
- [x]  SourceTypeService: add #forBlob()
- [x]  SourceTypeService methods should return a SourceTypeResult object
- [x] refactor import reducer and action
- [x] refactor DndImportPanel
- [x]  refactor ImportPlugin

replaces #701

### CODE REVIEW

**Comment by thiloSchlemmer** [comment]
fixes #727

File: src/modules/dndImport/components/DndImportPanel.js [hunk 1]
```diff
@@ -110,25 +110,29 @@ export class DndImportPanel extends MvuElement {
...
+							importData(f, sourceTypeResult.sourceType);
+							break;
+						case SourceTypeResultStatus.MAX_SIZE_EXCEEDED:
+							emitNotification(translate('dndImport_import_max_size_exceeded'), LevelTypes.WARN);
+							break;
+						case SourceTypeResultStatus.UNSUPPORTED_TYPE:
+							emitNotification(translate('dndImport_import_unsupported'), LevelTypes.WARN);
+							break;
+						case SourceTypeResultStatus.OTHER:
+							emitNotification(translate('dndImport_import_unknown'), LevelTypes.ERROR);
```
Comment (taulinger) [inline src/modules/dndImport/components/DndImportPanel.js:134]: just as a suggestion: can we combine sourceTypeResult.status handling within one block of code for all SourceTypeService calls?

File: src/modules/dndImport/components/DndImportPanel.js [hunk 2]
```diff
@@ -5,14 +5,13 @@ import { emitNotification, LevelTypes } from '../../../store/notifications/notif
 import css from './dndImportPanel.css';
```
[the remark this answers is not in the document]
Comment (thiloSchlemmer) [reply src/modules/dndImport/components/DndImportPanel.js:5]: Sure, thanks for the hint!

File: src/modules/dndImport/components/DndImportPanel.js (hunk 1 above)
Comment (thiloSchlemmer) [reply src/modules/dndImport/components/DndImportPanel.js:134]: Thx

**Review by taulinger — COMMENTED** [review]
Finally, let's do a last review and then go for merging...

**Review by thiloSchlemmer — APPROVED** [review]
Thx

**Comment by taulinger** [comment]
@thiloSchlemmer After merging close #701, please

**Pull request merged**